core/roboconf-core/src/main/java/net/roboconf/core/utils/Utils.java

```diff
@@ -120,33 +120,45 @@ public static boolean areEqual( Object o1, Object o2 ) {
 		return result;
 	}
 
+
 	/**
 	 * Expand a template, replacing each {{ param }} by the corresponding value.
+	 * <p>
 	 * Eg. "My name is {{ name }}" will result in "My name is Bond", provided that "params" contains "name=Bond".
-	 * @param s The template to expand
-	 * @param params The parameters to be expanded in the template
-	 * @return The expanded template.
+	 * </p>
+	 *
+	 * @param s the template to expand
+	 * @param params the parameters to be expanded in the template
+	 * @return the expanded template
 	 */
 	public static String expandTemplate(String s, Properties params) {
-		if(params == null || params.size() < 1) return s;
 
-		Pattern pattern = Pattern.compile( "\\{\\{\\s*\\S+\\s*\\}\\}" );
-		Matcher m = pattern.matcher(s);
+		String result;
+		if( params == null || params.size() < 1 ) {
+			result = s;
 
-		StringBuffer sb = new StringBuffer();
```

Comment:
When it works, why "fix" it ? (just to add more lines of code for stats ?? it does exactly the same thing than before... with 4 lines more).